data/menu.py

```diff
@@ -23,14 +23,10 @@
 
 # Main menu
 MENU = [
-    ('', 'Home'),
     ('news', 'News'),
-    ('security/', 'Security'),
```

Comment:
I think we should keep the Security link at the main menu.

Reply:
My first idea was to move "Security" to the "Support" submenu, but when I was implementing I saw that it will be adding more work than I was planning and move out for this. We can try to go back to this idea again.

Stated [here](https://github.com/phpmyadmin/website/issues/92#issuecomment-713899056):
> Moved "Security" to the "Support" submenu. "Security" as a (main) menu gives the idea that the product has a lot of Security Flaws. If we have it on "Support", gives the idea for the people who search for some support that we have attention to security, but for the main public doesn't need to have that attention, because it's the default behavior (to design a safe system).